src/components/EnvironmentSelection.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Alert } from 'react-native';
+import { SegmentedButtons, Text } from 'react-native-paper';
+import { createStyles } from './BrandConfigProvider';
+import { useAsyncStorage } from '../hooks/useAsyncStorage';
+import { View } from 'react-native';
+import { emitEnvironmentToggled, useStyles } from 'src/hooks';
+
+/*
+    Debug menu for selecting a secondary environment during runtime.
+    See AppDemo.tsx for implementation. Due to caching the app will require a manual
+    reload (via metro) after switching.
+*/
+
+export function EnvironmentSelection() {
+  const [storedEnvironment, setStoredEnvironment] =
+    useAsyncStorage('environment-toggle');
+  const [selectedButton, setSelectedButton] = useState('primary');
+  const { styles } = useStyles(defaultStyles);
+
+  const selectEnvironment = useCallback(
+    (value: string) => {
+      setStoredEnvironment(value);
+      setSelectedButton(value);
+      Alert.alert(
+        `Switched to ${value} environment`,
+        'To ensure new content is loaded perform a manual refresh of the app now.',
```

Comment:
Might just consider doing the refresh here if this is absolutely necessary: https://reactnative.dev/docs/devsettings#reload

Reply:
Added auto refresh (which I think is fine since this is a dev toggle).